providers/song-controls.js

```diff
@@ -12,8 +12,7 @@ module.exports = (win) => {
 		// Playback
 		previous: () => pressKey(win, "k"),
 		next: () => pressKey(win, "j"),
-		playPause: () => win.webContents.send("playPause"),
-		like: () => pressKey(win, "_"),
```

Comment:
Is it expected this one is removed?

Reply:
It actually has nothing to do with this pull - but the `like` function was written twice?
https://github.com/th-ch/youtube-music/blob/e0cb1326868594d172668712da8581b893c925ec/providers/song-controls.js#L28
https://github.com/th-ch/youtube-music/blob/e0cb1326868594d172668712da8581b893c925ec/providers/song-controls.js#L16

I just assumed it was a mistake since the first one works